portal/guides/connect-wallet.mdx

```diff
@@ -92,10 +68,7 @@ So make sure you put your `Component` within the brackets of the `ThirdwebProvid
 
 ```jsx:App.js
 	return (
-    <ThirdwebProvider
-      connectors={connectors}
-      supportedChainIds={supportedChainIds}
-    >
+    <ThirdwebProvider desiredChainId={supportedChainIds}>
       <Connect /> /* our connect wallet button */
```

Comment:
```suggestion
      <Connect /> /* our connect wallet button */
```
that's a syntax error, not a valid way of commenting in react

Reply:
adjusted